schemas/deviceToCloud/gnss/gnss.json

```diff
@@ -0,0 +1,100 @@
+{
+   "title": "GNSS",
+   "description": "GNSS device coordinate data",
+   "type": "object",
+   "oneOf": [
+      {
+         "properties": {
```

Comment:
**problem:** all properties are optional.

Reply:
The `required` prop at the bottom covers both items in the `oneOf` options array.